src/frontends/onnx/frontend/src/op/clip.cpp

```diff
@@ -29,6 +29,45 @@ OutputVector clip(const Node& node) {
 }  // namespace set_1
 
 namespace set_11 {
+namespace {
+std::shared_ptr<ov::op::v0::Constant> get_constant_lowest_of_type(ov::element::Type_t t) {
+#define OPENVINO_TYPE_TO_LOWEST_CONST(t)                                                       \
+    case t:                                                                                    \
+        return ov::op::v0::Constant::create(                                                   \
+            t,                                                                                 \
+            {},                                                                                \
+            {std::numeric_limits<typename ov::element_type_traits<t>::value_type>::lowest()}); \
+        break
+
+    switch (t) {
+        OPENVINO_TYPE_TO_LOWEST_CONST(ov::element::boolean);
+        OPENVINO_TYPE_TO_LOWEST_CONST(ov::element::bf16);
+        OPENVINO_TYPE_TO_LOWEST_CONST(ov::element::f16);
+        OPENVINO_TYPE_TO_LOWEST_CONST(ov::element::f32);
+        OPENVINO_TYPE_TO_LOWEST_CONST(ov::element::f64);
+        OPENVINO_TYPE_TO_LOWEST_CONST(ov::element::i8);
+        OPENVINO_TYPE_TO_LOWEST_CONST(ov::element::i16);
+        OPENVINO_TYPE_TO_LOWEST_CONST(ov::element::i32);
+        OPENVINO_TYPE_TO_LOWEST_CONST(ov::element::i64);
+        OPENVINO_TYPE_TO_LOWEST_CONST(ov::element::u1);
+        OPENVINO_TYPE_TO_LOWEST_CONST(ov::element::u8);
+        OPENVINO_TYPE_TO_LOWEST_CONST(ov::element::u16);
+        OPENVINO_TYPE_TO_LOWEST_CONST(ov::element::u32);
+        OPENVINO_TYPE_TO_LOWEST_CONST(ov::element::u64);
+
+    case ov::element::undefined:
+    case ov::element::dynamic:
+    default:
+        return nullptr;
+    }
+}
```

Comment:
May it be replaced with `make_tensor_of_min_value`?

Reply:
No, `lowest` returns different values for floating point type than `min`.